docker/templates/new-wagtail-app/{{ cookiecutter.app_name }}/Dockerfile

```diff
@@ -56,4 +56,4 @@ ENV DJANGO_DEBUG 'False'
 
 # Build static files into the container
 RUN python manage.py collectstatic --noinput
-RUN python manage.py compress
```

Comment:
this was failing with this error. I don't know why, but it's unrelated to the django-environ stuff.

```console
 > [app 12/12] RUN python manage.py compress:
0.668 Traceback (most recent call last):
0.668   File "/app/manage.py", line 22, in <module>
0.668     main()
0.668   File "/app/manage.py", line 18, in main
0.668 AWS config not found, defaulting to local storage
0.668 Compressing... Invalid template project/base.html: Invalid block tag on line 50: 'raw'. Did you forget to register or load this tag?
0.668 Error parsing template project/static_page.html: Invalid block tag on line 50: 'raw'. Did you forget to register or load this tag?
0.668 Error parsing template project/500.html: Invalid block tag on line 50: 'raw'. Did you forget to register or load this tag?
0.668 Error parsing template project/404.html: Invalid block tag on line 50: 'raw'. Did you forget to register or load this tag?
0.668 Error parsing template project/home_page.html: Invalid block tag on line 50: 'raw'. Did you forget to register or load this tag?
0.668 Error parsing template project/react_page.html: Invalid block tag on line 50: 'raw'. Did you forget to register or load this tag?
0.668     execute_from_command_line(sys.argv)
0.668   File "/usr/local/lib/python3.11/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line
0.668     utility.execute()
0.668   File "/usr/local/lib/python3.11/site-packages/django/core/management/__init__.py", line 436, in execute
0.668     self.fetch_command(subcommand).run_from_argv(self.argv)
0.668   File "/usr/local/lib/python3.11/site-packages/django/core/management/base.py", line 413, in run_from_argv
0.669     self.execute(*args, **cmd_options)
0.669   File "/usr/local/lib/python3.11/site-packages/django/core/management/base.py", line 459, in execute
0.669     output = self.handle(*args, **options)
0.669              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
0.669   File "/usr/local/lib/python3.11/site-packages/compressor/management/commands/compress.py", line 370, in handle
0.669     self.handle_inner(**options)
0.669   File "/usr/local/lib/python3.11/site-packages/compressor/management/commands/compress.py", line 395, in handle_inner
0.669     offline_manifest, block_count, results = self.compress(
0.669                                              ^^^^^^^^^^^^^^
0.670   File "/usr/local/lib/python3.11/site-packages/compressor/management/commands/compress.py", line 296, in compress
0.670     raise OfflineGenerationError(
0.670 compressor.exceptions.OfflineGenerationError: No 'compress' template tags found in templates.Try running compress command with --follow-links and/or--extension=EXTENSIONS
------
```

Reply:
`python manage.py compress` was for `django-compressor` but we're using Webpack now. This line should have been removed a while ago.

Reply:
removed.